src/partials/auth-security.md

```diff
@@ -24,4 +24,12 @@ If an user is imported into Appwrite with hash differnt than Argon2, the passwor
 Encourage passwords that are hard to guess by disallowing users to pick passwords that contain personal data.
 Personal data includes the user's name, email, and phone number.
 
-Disallowing personal data can be enabled in the Auth service's **Security** tab on the Appwrite Console.
\ No newline at end of file
+Disallowing personal data can be enabled in the Auth service's **Security** tab on the Appwrite Console.
+
+# Session alerts {% #session-alerts %}
+
+Enable email alerts for your users so that whenever another session is created for their account, they will be alerted to the new session.
+
+You won't receive notifications when logging in using [Magic URL](/docs/products/auth/magic-url), [Email OTP](/docs/products/auth/email-otp), or [OAuth2](/docs/products/auth/oauth2) since these authentication methods already verify user access to their systems, establishing the authentication's legitimacy.
+
+To toggle session alerts, navigate to **Auth** > **Security** > Session Alerts.
```

Comment:
Let's bold session alerts since [all text seen as-is in the UI should be bold](https://github.com/appwrite/website/blob/7a7f74f4c6acba1937505aa989f5626867e272e7/CONTENT.md?plain=1#L110) and make it sentence case so it matches the Console.

```suggestion
To toggle session alerts, navigate to **Auth** > **Security** > **Session alerts**.
```